### PR TITLE
🌐 Fix Korean translation for `docs/ko/docs/tutorial/index.md`

### PR DESCRIPTION
**Pull request by kimjaeyoonn**
Hello. This PR is an updated document to correct some inconveniences in #2317.

@hard-coders 
Please check a few updated sentences, and if you find it appropriate, please leave a positive comment.
Additionally, it would be much appreciated if you could suggest a merge to tiangolo.

This PR complements `ko/docs/tutorial/index.md`
related : #2017

### CODE REVIEW

**Review by kty4119 — APPROVED** [review]

**Review by DevDae — APPROVED** [review]
Appreciate your works! 🚀 
It's better to read the docs.

File: docs/ko/docs/tutorial/index.md [hunk 1]
```diff
@@ -2,11 +2,11 @@
 
 이 자습서는 **FastAPI**의 대부분의 기능을 단계별로 사용하는 방법을 보여줍니다.
 
-각 섹션은 이전 섹션을 기반해서 점진적으로 만들어 졌지만, 주제를 구분하여 구성 되었기 때문에 특정 API 요구사항을 해결하기 위해 어떤 특정 항목이던지 직접 이동할 수 있습니다.
+각 섹션은 이전 섹션을 기반으로 하여 점진적으로 만들어 졌지만, 주제에 따라 다르게 구성되었기 때문에 특정 API 요구사항을 해결하기 위해서라면 어느 특정 항목으로던지 직접 이동할 수 있습니다.
 
-또한 향후 참조가 될 수 있도록 만들어졌습니다.
+추가적으로 이는 향후 참조가 될 수 있도록 만들어졌습니다.
```
Comment (hard-coders) [inline docs/ko/docs/tutorial/index.md:7]: - 기반해서 -> 기반으로 하여: 둘이 완전히 동일한 뜻이므로 굳이 풀어서 사용할 이유는 없어 보입니다.
- 또한 -> 추각적으로 이는: 너무 직역체입니다.

**Review by hard-coders — COMMENTED** [review]
Thanks for your feedback! It's more natural.
I left some of my opinions. Would you mind checking and telling me about your thought?

File: docs/ko/docs/tutorial/index.md (hunk 1 above)
Comment (kimjaeyoonn) [reply docs/ko/docs/tutorial/index.md]: Thanks for posting a reply!!
The documentation has been updated to reflect your additional comments.
I hope the updated documents will be merged🚀

**Review by hard-coders — APPROVED** [review]
LGTM 🚀

**Comment by tiangolo** [comment]
Awesome, thanks for the contribution @kimjaeyoonn! :bow: 

And thanks for the reviews @hard-coders, @kty4119, @DevDae! :clap:

**Pull request merged**